src/app/pages/system/bootenv/bootenv-status/bootenv-status.component.ts

```diff
@@ -101,96 +104,105 @@ export class BootStatusListComponent implements OnInit {
     if (data.stats) {
       stats = data.stats;
     }
+
+    let name: string;
```

Comment:
@undsoft 
![image](https://user-images.githubusercontent.com/22980553/124500844-b09b8500-ddea-11eb-951d-4f3115bd8a7e.png)
we need to set the initial value for 'name', e.g. let name = data.name

Reply:
Good catch. Fixed.